images/linux/scripts/installers/azure-cli.sh

```diff
@@ -6,5 +6,8 @@
 
 # Install Azure CLI (instructions taken from https://docs.microsoft.com/en-us/cli/azure/install-azure-cli)
 curl -sL https://aka.ms/InstallAzureCLIDeb | sudo bash
+echo "azure-cli https://docs.microsoft.com/en-us/cli/azure/install-azure-cli-linux?pivots=apt" >> $HELPER_SCRIPTS/apt-sources.txt
```

Comment:
Should it be `https://packages.microsoft.com/repos/azure-cli/`?

Reply:
@miketimofeev i think no. The URL you've mentioned is not documented well and the shell script to add the microsoft provided repo is the recommended way. This way i put a link to the documentation page describing the way we've installed azure cli and an user should use if he wants to (re)install azure-cli